coins/ethereum/src/blockchain_client.rs

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
This blockchain_client should not be in a change marked as additional tests for KeyPairBuilder.